drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -874,24 +874,27 @@ static void vc4_hdmi_recenter_fifo(struct vc4_hdmi *vc4_hdmi)
 {
 	u32 drift;
 	int ret;
+	int i;
 
-	drift = HDMI_READ(HDMI_FIFO_CTL);
-	drift &= VC4_HDMI_FIFO_VALID_WRITE_MASK;
-
-	HDMI_WRITE(HDMI_FIFO_CTL,
-		   drift & ~VC4_HDMI_FIFO_CTL_RECENTER);
-	HDMI_WRITE(HDMI_FIFO_CTL,
-		   drift | VC4_HDMI_FIFO_CTL_RECENTER);
-	usleep_range(1000, 1100);
-	HDMI_WRITE(HDMI_FIFO_CTL,
-		   drift & ~VC4_HDMI_FIFO_CTL_RECENTER);
-	HDMI_WRITE(HDMI_FIFO_CTL,
-		   drift | VC4_HDMI_FIFO_CTL_RECENTER);
-
-	ret = wait_for(HDMI_READ(HDMI_FIFO_CTL) &
-		       VC4_HDMI_FIFO_CTL_RECENTER_DONE, 1);
-	WARN_ONCE(ret, "Timeout waiting for "
-		  "VC4_HDMI_FIFO_CTL_RECENTER_DONE");
+	for (i = 0; i < 0; i++) { // Once again, with feeling
```

Comment:
That's an odd looking loop. It looks more like "Really, don't bother"!

Reply:
hmm, must have pressed undo before merging. It probably is required but I guess the other changes are the sigificant ones